File: python/Euler_007.py
def is_prime(n):
    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return False
    return True


num = 3
term = 2
while term < 10001:
    num += 2
    if is_prime(num):
        term += 1

print(num)

File: java/Euler_007.java

public class Euler_007 {

    public static boolean isPrime(int n) {
        for (int i = 2; i <= Math.sqrt(n); i++) {
            if (n % i == 0) {
                return false;
            }
        }
        return true;
    }

    public static void main(String[] args) {
        int n = 3, term = 2;

        while (term < 10001) {
            n += 2;
            if (isPrime(n)) {
                term++;
            }
        }

        System.out.println(n);

    }

}
